### PR TITLE
Added feroxbuster and weevely tools.

### PR DESCRIPTION
**Pull request by Fastiraz**
I've only added the feroxbuster and weevely tools to the cheats.

Feroxbuster is a fast, modern, simple, recursive content discovery tool written in Rust.

Weevely is a stealth PHP web shell that simulate telnet-like connection. It is an essential tool for web application post exploitation, and can be used as stealth backdoor or as a web shell to manage legit web accounts, even free hosted ones.

< Happy hacking! />

### CODE REVIEW

**Comment by Guilhem7** [comment]
Hi @Fastiraz,
Thanks for your help !

We would like to suggest some changes before merging your commits.
For the commit about __feroxbuster__:
- Please do not use local path in the cheat: *./feroxbuster* --> *feroxbuster* 
- Reduce the number of cheats for the tool, those you think are useful. (For instance the option: *--threads 200* can be remove in my opinion)

Use args from user:
```bash
feroxbuster -u http://127.1 --burp
```
To
```bash
feroxbuster -u <url> --burp
```

Also for the commit about __weevely__:
 - Remove the help *weevely generate \<password\>*, as this command does nothing

Use default arguments if needed:
```
weevely generate <password> web-shell.php
```
To
```
weevely generate <password> <output_file|web_shell.php>
```
Thanks again

**Comment by Fastiraz** [comment]
I just made the last commits following your instructions.
Thanks for your feedback.

File: arsenal/data/cheats/Web/fuzzer.md [hunk 1]
```diff
@@ -84,4 +84,34 @@ ffuf -w <wordlist> -u <url> -X POST -d "username=admin\&password=FUZZ" -fc 401
...
-= wordlist: /usr/share/wordlists/dirbuster/directory-list-2.3-medium.txt
\ No newline at end of file
+# feroxbuster
+
+% fuzzer, fuzz, ffuf, dirsearch, gobuster, dirb
+#plateform/linux #target/remote #cat/ATTACK/FUZZ
+
+## default scan
+```
+feroxbuster --url <URL>
```
Comment (Guilhem7) [inline arsenal/data/cheats/Web/fuzzer.md:94]:
```suggestion
feroxbuster --url <url>
```
We might prefer lowercase for variable name :)

File: arsenal/data/cheats/Web/fuzzer.md [hunk 2]
```diff
@@ -84,4 +84,34 @@ ffuf -w <wordlist> -u <url> -X POST -d "username=admin\&password=FUZZ" -fc 401
...
+```
+
+## default scan with wordlist
+```
+feroxbuster --url <URL> -w <wordlist>
```
Comment (Guilhem7) [inline arsenal/data/cheats/Web/fuzzer.md:99]:
```suggestion
feroxbuster --url <url> -w <wordlist>
```

File: arsenal/data/cheats/Web/fuzzer.md [hunk 3]
```diff
@@ -84,4 +84,34 @@ ffuf -w <wordlist> -u <url> -X POST -d "username=admin\&password=FUZZ" -fc 401
...
+```
+
+## Multiple headers:
+```
+feroxbuster -u <URL> -H Accept:application/json "Authorization: Bearer {token}"
```
Comment (Guilhem7) [inline arsenal/data/cheats/Web/fuzzer.md:104]:
```suggestion
feroxbuster -u <url> -H "<header>" "<header>"
```

File: arsenal/data/cheats/Web/fuzzer.md [hunk 4]
```diff
@@ -84,4 +84,34 @@ ffuf -w <wordlist> -u <url> -X POST -d "username=admin\&password=FUZZ" -fc 401
...
+```
+feroxbuster --url <URL>
+```
+
+## default scan with wordlist
+```
+feroxbuster --url <URL> -w <wordlist>
+```
+
+## Multiple headers:
```
Comment (Guilhem7) [inline arsenal/data/cheats/Web/fuzzer.md:102]:
```suggestion
## Multiple headers
```

File: arsenal/data/cheats/Web/fuzzer.md [hunk 5]
```diff
@@ -84,4 +84,34 @@ ffuf -w <wordlist> -u <url> -X POST -d "username=admin\&password=FUZZ" -fc 401
...
+```
+
+## IPv6, non-recursive scan with INFO-level logging enabled:
+```
+feroxbuster -u http://[::1] --no-recursion -vv
+```
+        
+## Abort or reduce scan speed to individual directory scans when too many errors have occurred
+```
+feroxbuster -u <URL> --auto-bail
```
Comment (Guilhem7) [inline arsenal/data/cheats/Web/fuzzer.md:114]:
```suggestion
feroxbuster -u <url> --auto-bail
```

File: arsenal/data/cheats/Web/fuzzer.md [hunk 6]
```diff
@@ -84,4 +84,34 @@ ffuf -w <wordlist> -u <url> -X POST -d "username=admin\&password=FUZZ" -fc 401
...
+```
+
+## Multiple headers:
+```
+feroxbuster -u <URL> -H Accept:application/json "Authorization: Bearer {token}"
+```
+
+## IPv6, non-recursive scan with INFO-level logging enabled:
+```
+feroxbuster -u http://[::1] --no-recursion -vv
```
Comment (Guilhem7) [inline arsenal/data/cheats/Web/fuzzer.md:109]:
```suggestion
feroxbuster -u <proto|https>://[<ipv6>] --no-recursion -vv
```
Let the user choose the protocol (default https I think), and ipv6

File: arsenal/data/cheats/Web/fuzzer.md [hunk 7]
```diff
@@ -84,4 +84,34 @@ ffuf -w <wordlist> -u <url> -X POST -d "username=admin\&password=FUZZ" -fc 401
...
+```
+feroxbuster --url <URL> -w <wordlist>
+```
+
+## Multiple headers:
+```
+feroxbuster -u <URL> -H Accept:application/json "Authorization: Bearer {token}"
+```
+
+## IPv6, non-recursive scan with INFO-level logging enabled:
```
Comment (Guilhem7) [inline arsenal/data/cheats/Web/fuzzer.md:107]:
```suggestion
## IPv6, non-recursive scan with INFO-level logging enabled
```

**Review by Guilhem7 — CHANGES_REQUESTED** [review]
Hi, thanks for the modifications, I added some little details to change if possible :)

**Comment by Fastiraz** [comment]
Feroxbuster done, is there anything to edit in weevely file ?

**Review by Guilhem7 — APPROVED** [review]

**Comment by Guilhem7** [comment]
Hey,
everything seems good to me, thanks for the modifs and for the PR !

**Pull request merged**